live/standard/eu-west-2/full/physical/terragrunt.hcl

```diff
@@ -0,0 +1,20 @@
+# Terragrunt will copy the Terraform configurations specified by the source parameter, along with any files in the
+# working directory, into a temporary folder, and execute your Terraform commands in that folder.
+terraform {
+  source = "../../../../../cloudprem//physical"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+retryable_errors = [
+  "(?s).*error waiting for Route in Route Table.*waiting for state to become 'ready'.*"
+]
+
+# These are the variables we have to pass in to use the module specified in the terragrunt configuration above
+inputs = {
+
+}
+
+skip = get_env("SKIP_FULL", false)
```

Comment:
Not a blocker, more of a question: Can this be moved up top like some of the other skip values in some of the other files?

Like in `live/standard/eu-west-2/full/logical/terragrunt.hcl`

Reply:
Sure